Buildings/Electrical/AC/ThreePhasesBalanced/Loads/MotorDrive/Coupled/Pump.mo

```diff
@@ -0,0 +1,228 @@
+    "Record of pump with performance data"
+     annotation (choicesAllMatching=true,Placement(transformation(extent={{-80,-68},{-60,-48}})));
+
+  //Motor parameters
+  replaceable parameter Buildings.Electrical.AC.ThreePhasesBalanced.Loads.MotorDrive.InductionMotors.Data.Generic
+    per1 constrainedby Buildings.Electrical.AC.ThreePhasesBalanced.Loads.MotorDrive.InductionMotors.Data.Generic
+    "Record of Induction Machine with performance data"
+    annotation (Dialog(tab="Motor"), choicesAllMatching=true, Placement(transformation(extent={{52,60},{72,80}})));
+  parameter Boolean have_controller = true
+    "Set to true for enableing motor PID control"
+    annotation (Dialog(tab="Motor"));
+  parameter Modelica.Blocks.Types.SimpleController controllerType=Modelica.Blocks.Types.SimpleController.PI
+    "Type of controller"
+    annotation (Dialog(tab="Motor", group="Motor controller", enable=have_controller));
+  parameter Real k(min=0) = 1
+    "Gain of controller"
+    annotation (Dialog(tab="Motor", group="Motor controller", enable=have_controller));
+  parameter Modelica.Units.SI.Time Ti(min=Modelica.Constants.small)=0.5
+    "Time constant of Integrator block"
+    annotation (Dialog(tab="Motor", group="Motor controller",
+                       enable=have_controller and
+                              controllerType == Modelica.Blocks.Types.SimpleController.PI or
+                              controllerType == Modelica.Blocks.Types.SimpleController.PID));
+  parameter Modelica.Units.SI.Time Td(min=0) = 0.1
+    "Time constant of Derivative block"
+    annotation (Dialog(tab="Motor", group="Motor controller",
+                       enable=have_controller and
+                              controllerType == Modelica.Blocks.Types.SimpleController.PD or
+                              controllerType == Modelica.Blocks.Types.SimpleController.PID));
+  parameter Real yMax(start=1)=1
+    "Upper limit of output"
+    annotation (Dialog(tab="Motor", group="Motor controller", enable=have_controller));
+  parameter Real yMin=0
+    "Lower limit of output"
+    annotation (Dialog(tab="Motor", group="Motor controller", enable=have_controller));
+
+  parameter Modelica.Units.SI.Inertia loaIne=1 "Pump inertia"
+    annotation (Dialog(group="Motor"));
+  parameter Modelica.Units.NonSI.AngularVelocity_rpm Nrpm_nominal=1500
+    "Nominal rotational speed of pump" annotation (Dialog(group="Motor"));
+  parameter Modelica.Fluid.Types.Dynamics energyDynamics=Modelica.Fluid.Types.Dynamics.DynamicFreeInitial
+    "Type of energy balance: dynamic (3 initialization options) or steady state"
+    annotation (Dialog(tab="Dynamics", group="Conservation equations"));
+  parameter Modelica.Units.SI.Time tau=1
+    "Time constant of fluid volume for nominal flow, used if energy or mass balance is dynamic"
+    annotation (Dialog(tab="Dynamics", group="Conservation equations"));
+  parameter Boolean use_riseTime=true
```

Comment:
Will the rise time affect the motor rotation?

Reply:
How does the rise time and motor rotation affect each other? Is it possible to add the clarification into the documentation? User would like to know how to set it and what will happen if the rise time changes.